_posts/2019-06-16-good-first-issue.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "goodfirstissue - what, why and how?"
+description: In this post Rajat will share what triggered him to write goodfirstissue app, where was it deployed and how it was a nobrainer to move to OpenFaaS cloud.
+date: 2019-06-16
+image: /images/good-first-issue/anukrati-omar-1206574-unsplash.jpg
+categories:
+  - templates
+  - golang
+  - go
+author_staff_member: rajatjindal
+dark_background: true
+---
+I’d like to introduce you to Rajat Jindal who is our guest-writer for today’s end-user blog post. Rajat will talk about the goodfirstissue bot that he wrote for helping connect first time contributors with open source projects that encourage contributions from first timers.
+
+> Author bio: [Rajat Jindal](https://twitter.com/rajatjindal1983?lang=en-gb) is staff software engineer at a US based cyber security company. He’s working on service delivery team making tools for developers to help them ship their services to production with high quality and velocity.
+
+<img src="/images/good-first-issue/rajat.jpg" width="25%" height="25%" />
+
+### What is a "good first issue"?
```

Comment:
Can you try to restructure a little please so that you do the following:

1) Define the problem as you see it
2) Talk about the opportunity
3) Show how you solved the problem
4) Show how people are benefitting from the solution you built
5) Give credit anyone who has helped you and any of the feedback you've had from users
6) Talk about any limitations, opportunities to grow, future plans and direction

Reply:
Thoughts on this @rajatjindal ?

Reply:
Points 4 + 5 + 6 were not addressed in the original post, have you had a chance to address that part now?

Reply:
not yet, i will address these. There were some confusions that I wanted to clarify with you on chat: 

with point # 1, 2 and 3. I think we cover them already in the post or i am misunderstanding something.

for # 5, I got confused by this from README file

```
    Acknowledgements

If necessary show acknowledgements to others who collaborated or gave feedback you 
used. When working on GitHub this is available in public history, so use sparingly.

```

I do have a few items for # 6 so I will update the post to reflect them.

Thanks for your review feedback

Reply:
Please add the amendments you have and then I'll check if 1, 2 and 3 are covered 👍

Reply:
Hi Alex, I've added the changes to address to all the above points. Kindly let me know if more structural changes are needed.